hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -889,12 +890,19 @@ def magic(
         raise InvalidArgument("Must pass at least one function or module to test.")
     functions = set()
     for thing in modules_or_functions:
-        if callable(thing):
+        if inspect.isclass(thing):
```

Comment:
I'd switch these; `if callable(thing):`/`elif inspect.isclass(thing):` - it's mostly an aesthetic sense that if we *can* treat it like a function, that's *probably* what makes sense here.

Especially since we'll then pick up the other cases down belowon line 922.

Reply:
We don't even need the `elif inspect.isclass(thing):` if we include the class as a callable